sdk/cosmos/microsoft-azure-cosmos-benchmark/pom.xml

```diff
@@ -6,14 +6,14 @@ Licensed under the MIT License.
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.microsoft.azure</groupId>
+    <groupId>com.azure</groupId>
```

Comment:
As this transitions from `com.microsoft.azure` (track one) to `com.azure` (track two), the directory naming should also transition from `microsoft-azure-X` to `azure-X`.

Reply:
Done